daos/Tuit2TagDao.ts

```diff
@@ -45,7 +45,7 @@ export default class Tuit2TagDao implements Tuit2TagDaoI {
         Tuit2TagModel.deleteOne({tuit: tuitID, tag: tagID});
```

Comment:
I think we should use "deleteMany" instead of "deleteOne" here for the off-chance a user adds two of the same tag to one tuit.